js/formHandler.js

```diff
@@ -0,0 +1,136 @@
+let form = document.querySelector('.currency_rate_form');
+
+let currRate = new CurrencyRate();
+
+form.onsubmit = () => {
+
+
+    let from = document.querySelector('#dateFrom').value;
+    let to = document.querySelector('#dateTo').value;
+
+    let dateFrom = new Date(from);
+    let dateTo = new Date(to);
+    if(dateFrom > dateTo){
```

Comment:
will be not obvious for user that you switch dates, the validation should be done on the input level and the appropriate message should be shown to user